pkg/workspace/tuning/preset-tuning.go

```diff
@@ -256,16 +267,17 @@ func CreatePresetTuning(ctx context.Context, workspaceObj *kaitov1beta1.Workspac
 	return jobObj, nil
 }
 
-// Now there are two options for data destination 1. HostPath - 2. Image
-func prepareDataDestination(ctx context.Context, workspaceObj *kaitov1beta1.Workspace, inputDirectory string) (*corev1.Container, *corev1.LocalObjectReference, corev1.Volume, corev1.VolumeMount) {
+// Now there are two options for data destination 1. Volume - 2. Image
+func prepareDataDestination(ctx context.Context, workspaceObj *kaitov1beta1.Workspace, inputDirectory string) (*corev1.Container, *corev1.LocalObjectReference, []corev1.Volume, []corev1.VolumeMount) {
 	tuning := workspaceObj.Tuning
 	output := tuning.Output
 
-	outputImage := output.Image
-	if outputImage == "" {
-		return nil, nil, corev1.Volume{}, corev1.VolumeMount{}
+	if output.Volume != nil {
+		return nil, nil, nil, nil
 	}
 
+	outputImage := output.Image
+
```

Comment:
why we delete `if outputImage == ""` here, if image is empty, since we don't need to add a sidecar image in L202 but only need a pause image like code before

Reply:
resolved